applications/callflow/src/module/cf_park.erl

```diff
@@ -197,6 +198,7 @@ park_call(SlotNumber, Slot, ParkedCalls, ReferredTo, Call) ->
         %% attended transfer and allowed to update the provided slot number, we are still connected to the 'parker'
         %% not the 'parkee'
         {'undefined', _} ->
+            _ = publish_parked(Call, SlotNumber),
```

Comment:
@macpie 
Luis is correct, this is not yet parked.  There is a function to handle the completion of an attended transfer and swap the caller-id, that would likely be more appropriate for this.